en/docs/tutorials/integrating-services-tutorial.md

```diff
@@ -0,0 +1,795 @@
+- Generating a response for each order request based on the status (i.e., whether the requested item is available and whether the available quantity is adequate to match the quantity requested).
+
+To create this service, follow the procedure below:
+
+1. Access the Choreo Console and click **Services**. Then click **Create**.
+
+2. Enter `inventory` as the name and click **Create**.
+
+3. To define the API that triggers your service, click **PUT**, enter `/inventory` in the **Path** field, and then click **Save API**.
+
+    ![Resource Configuration](../assets/img/tutorials/inventory-service-api-trigger.png){.cInlineImage-half}
+    
+4. To read the information in the order, define the required parameters.
+
+    In this scenario, the information derived from each order includes the name of the item ordered, and the quantity required.
+    
+    Therefore, let's define the parameters via variables as follows:
+    
+    1. First, define a variable to contain the variables that define the parameters.
+    
+        To do this,  click **Variable**.
+        
+        Enter the following information:
+        
+        | **Field**         | **Value**                  |
+        |-------------------|----------------------------|
+        | **Type**          | **other**                  |     
+        | **Other Type**    | `map<string[]>`            |
+        | **Name**          | `queryParams`              |
+        | **Expression**    | `request.getQueryParams()` |
+        
+        Click **Save**.
+
+    2. To allow the service to identify the name of the item ordered, define another variable.
+    
+        Click the **+** icon below the `queryParams`statement, and then click **Variable**.
+        
+        Enter the following information:
+    
+        | **Field**         | **Value**                    |
+        |-------------------|------------------------------|
+        | **Type**          | **string**                   |     
+        | **Name**          | `inventoryItemId`            |
+        | **Expression**    | `queryParams.get("item")[0]` |
+        
+        Click **Save**.
+        
+    3. To allow the service to identify the quantity ordered, define another variable.
+    
+        Click the **+** icon below the `inventoryItemId` variable statement, and then click **Variable**.
+        
+        Enter the following information:
+        
+        | **Field**         | **Value**                                           |
+        |-------------------|-----------------------------------------------------|
+        | **Type**          | **int**                                             |     
+        | **Name**          | `quantity`                                          |
+        | **Expression**    | `check int:fromString(queryParams.get("count")[0])` |
+        
+        Click **Save**.
+     
+ 5. To connect to the Google sheet where inventory records are maintained, add an API call as follows: 
```

Comment:
Shall we say why we need to connect Google sheet here? Like: `Inventory records are maintained in a Google Sheet. So we need to connect to that sheet from this Service to update the sheet. So to connect to the Google sheet where inventory records are maintained, add an API call as follows:`